### PR TITLE
Upgrade ios-simple to use Appium 1.6.0

### PR DESCRIPTION
**Pull request by dpgraham**
-Replaced elementByIosUIAutomation call with elementByAccessibilityId because we use the appium-xcui-test-driver which doesn't support UIAutomation
-Upgraded appium-version in desiredCapabilities config file (caps.js) to version 1.6

### CODE REVIEW

**Comment by jlipps** [comment]
@dpgraham it looks like your git user isn't set up with the same e-mail as your github user, mind fixing that and re-committing? code LGTM

**Comment by imurchie** [comment]
Moving forward can you put the language in the title, too (`ios-simple` exists in a number of the languages in this repo)? Looks good, otherwise!

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (jlipps) [inline .vscode/launch.json]: we probably don't want to commit visual studio code files. my suggestion would be to add this to your global `.gitignore`

**Comment by jlipps** [comment]
Oh, @dpgraham looks like your machine's github user is still not set up to use the same one as `@dpgraham` on github

**Comment by jlipps** [comment]
@dpgraham mind repushing as your GH user, and cleaning up the `launch.json` file? then we can merge this.

**Comment by dpgraham** [comment]
I've pushed to it with my new credentials.

**Comment by jlipps** [comment]
@dpgraham it's still not working; somehow your old user is also attached on some of these commits. it might be better to just squash them so it all gets rewritten as your new user, and force-push to the PR branch?

**Comment by dpgraham** [comment]
Done. I squashed them together and reworded the commit message. Should be good now.

**Pull request merged**